src/components/TokenSelection/TokenSelection.styles.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  background-color: ${(props) => props.theme.colors.black};
+  padding: 2rem;
+`;
+
+export const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 2rem;
+`;
+
+export const ArrowContainer = styled.div`
```

Comment:
```
export const BackButton = styled.button`
```

Also add some padding (0.5rem) because the button is really small.

I also would like to use the IconButton for this, but we can do that later.